airflow-core/tests/unit/dag_processing/test_collection.py

```diff
@@ -287,6 +288,7 @@ def test_add_asset_activate_already_exists(self, dag_maker, session):
         session.flush()
         assert orm_assets["myasset", "file://myasset/"].active is not None, "should pick up existing active"
 
+    @pytest.mark.xfail(reason="conflict not handled when serdag creation enabled")
```

Comment:
Is there an issue for this?

Reply:
yeah it was this one https://github.com/apache/airflow/issues/50405

i can now remove the xfail here also